acceptance/tests/loader/autoload_from_resource_type_decl.rb

```diff
@@ -4,80 +4,64 @@
   require 'puppet/acceptance/environment_utils.rb'
   extend Puppet::Acceptance::EnvironmentUtils
 
-  require 'master_manipulator'
-  extend MasterManipulator::Log
-
-  puppet_server_log      = puppet_server_log_path(master)
   app_type               = File.basename(__FILE__, '.*')
   tmp_environment        = mk_tmp_environment_with_teardown(master, app_type)
   fq_tmp_environmentpath = "#{environmentpath}/#{tmp_environment}"
 
   relative_type_dir    = 'modules/one/lib/puppet/type'
-  relative_type_1_path = "#{relative_type_dir}/type_tst.rb"
+  relative_type_path   = "#{relative_type_dir}/type_tst.rb"
+  relative_helper_dir  = "#{relative_type_dir}/helper"
+  relative_helper_path = "#{relative_helper_dir}/required_helper.rb"
   step 'create custom type' do
-    on(master, "mkdir -p #{fq_tmp_environmentpath}/#{relative_type_dir}")
+    on(master, "mkdir -p '#{fq_tmp_environmentpath}/#{relative_type_dir}' '#{fq_tmp_environmentpath}/#{relative_helper_dir}'")
 
     custom_type = <<-END
+    require 'puppet/type/helper/required_helper'
```

Comment:
Hmm - wasn't this supposed to be `require 'helper/required_helper'`?

Might want to note something like `# based on the masters $LOAD_PATH this Ruby will not be found. The generate command $LOAD_PATH can find the code however`